distserve/evaluation/docs/README-AE.md

```diff
@@ -179,20 +179,17 @@ bash /app/distserve/distserve/evaluation/ae-scripts/e2e/plot-fig-10.sh
 to generate Figure. 10. Plots will be saved under `/workspace/plots`.
 
 ### Ablation Studies (Section 6.4, Figure. 11)
-Follow the steps below to create a instance with one `RTX3090` GPU on [RunPod](https://www.runpod.io/) with template `DistLLM-AE-GPU`: 
-- Log in to [RunPod](https://www.runpod.io/) with the credentials provided in hotcrp.
-- Switch the account from `osdi24ae` to `Hao Lab@UCSD` using the upper right button.
-- Click `Pods` in the left toolbar.
-- Click `+ Deploy`.
-- Choose `RTX3090`.
-- Click `Change Template` and choose `DistLLM-AE-GPU`.
-- Choose `GPU Count`: 1 GPU is sufficient. In fact, the ablation experiment can be run on CPU instances, but the template we provide requires GPU environment.
-- Click `Deploy On-Demand`: If the button is grey, it means this resource is not currently available.
 
-TODO: @cjd plot to pdf instead of png
 
-Execute xxx
+*Compute Time: 5 min*
+
+The abalation study is CPU-only. We preferred you allocate `RTX3090` or `L40S` where 32 vCPU instance is available.
+
```

Comment:
"When you successfully log into the machine, execute"

Reply:
Changed the text